module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -138,6 +145,7 @@ class Manager implements
      *
      * @param Config         $config         VuFind configuration
      * @param UserTable      $userTable      User table gateway
+     * @param LoginToken     $loginToken     Login Token
```

Comment:
Is there a strong reason to add the new argument here in the list? Might it be better to add it to the end of the list for greater backward compatibility?

Reply:
Moved to the end of the list